lib/ocpp/v201/charge_point.cpp

```diff
@@ -75,6 +75,11 @@ ChargePoint::ChargePoint(const std::map<int32_t, int32_t>& evse_connector_struct
     }
 
     initialize(evse_connector_structure, message_log_path);
+
+    this->message_dispatcher =
```

Comment:
Might want to check here if `message_queue` exists because this constructor is sometimes called with nullptr.